scripts/requirements.txt

```diff
@@ -1,27 +1,27 @@
 Pillow
-PyYAML>=5.1
+PyYAML==5.3
```

Comment:
Can you please do this in a separate file (requirements-nrf-ci.txt or something) instead?

Reply:
I mean not just this line, but the entire commit, just to be explicit.

Reply:
I understand this will cause conflicts in the upmerge.    But these changes are not just for CI.  The floating versions cause an issue for customers also.

Reply:
removed this change, we will look at other tools to solve this issue after the release.   (pipenv?)